mcp-servers/README.md

```diff
@@ -1,479 +1,386 @@
+# Install Toolhive CRDs
+helm install toolhive-crds toolhive/toolhive-operator-crds --version 0.0.18
 
-# Port forward for local testing
-oc port-forward svc/mcp-weather 8000:8000
+# Install Toolhive operator with adequate memory resources
+helm install toolhive-operator toolhive/toolhive-operator --version 0.2.6 --namespace <your-namespace>
 
-# Test MCP weather server endpoint
-curl -X GET http://localhost:8000/health
-```
+# IMPORTANT: Increase memory limits to prevent OOMKilled errors
+oc patch deployment toolhive-operator --namespace <your-namespace> --type='json' \
+  -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/resources/limits/memory", "value": "1Gi"},
+       {"op": "replace", "path": "/spec/template/spec/containers/0/resources/requests/memory", "value": "1Gi"}]'
 
-### SSE Endpoint Testing
+# Grant required SecurityContextConstraints for Toolhive operator
+oc adm policy add-scc-to-user anyuid -z toolhive-operator --namespace <your-namespace>
 
-```bash
-# Connect to SSE endpoint
-curl -N -H "Accept: text/event-stream" \
-  http://localhost:8000/sse
-
-# Test weather query capability
-curl -X POST http://localhost:8000/tools/weather \
-  -H "Content-Type: application/json" \
-  -d '{
-    "location": "New York, NY",
-    "units": "metric"
-  }'
+# Verify Toolhive operator is running
+oc get pods -l app.kubernetes.io/name=toolhive-operator --namespace <your-namespace>
 ```
 
-### Integration with LlamaStack
-
-Configure LlamaStack to use MCP servers:
+**Step 4: Enable MCP Servers and Grant Required Permissions**
+```bash
+# Create configuration file to enable both MCP servers
+cat > mcp-config.yaml << EOF
+toolhive:
+  crds:
+    enabled: false  # CRDs already exist
+  operator:
+    enabled: true
 
-```yaml
-# In LlamaStack configuration
 mcp-servers:
-  weather-server:
-    endpoint: "http://mcp-weather:8000/sse"
-    capabilities:
-      - weather_lookup
-      - location_search
-    timeout: 30s
+  mcp-weather:
+    mcpserver:
+      enabled: true
+      env:
+        TAVILY_API_KEY: ""  # Add your API key if needed
+      permissionProfile:
+        name: network
+        type: builtin
+
+  oracle-sqlcl:
+    mcpserver:
+      enabled: true
+      env:
+        ORACLE_USER: "sales"  # Sales schema user created by Oracle DB chart
+        ORACLE_PASSWORD: null  # Sourced from secret
+        ORACLE_CONNECTION_STRING: null  # Sourced from secret
+        ORACLE_CONN_NAME: "oracle_connection"
+      envSecrets:
+        ORACLE_PASSWORD:
+          name: oracle23ai
+          key: password
+        ORACLE_CONNECTION_STRING:
+          name: oracle23ai
+          key: jdbc-uri
+      permissionProfile:
+        name: network
+        type: builtin
+    volumes:
+      - name: sqlcl-data
+        persistentVolumeClaim:
+          claimName: oracle-sqlcl-data
+    volumeMounts:
+      - name: sqlcl-data
+        mountPath: /sqlcl-home
+EOF
+
+# Create required PVC for Oracle SQLcl MCP server
```

Comment:
why not helm installation can do it? is there any reason to do it as additional step?